documentation/Theme: MyWebClass/Epics/Epics_1_MyWebClass.md

```diff
@@ -37,3 +37,5 @@ business value and impact, then it may be a lower priority.
 
 # List stories related to this theme
 1. [Story 1](../Stories/Story_1_MyWebClass.md)
+
```

Comment:
3. Make sure your team knows google analytics or else you would have to resolve more issues.